string command    #move, grasp, send
string target
string[] option1  #<grasp> on | <send> location
string[] option2  #<grasp> close | <send> close
